_data/krabhisharma/krabhisharma.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Abhishek Kumar 
+institution: THE ICFAI FOUNDATION FOR HIGHER EDUCATION,HYDERABAD 🚩 
+profile_pic: abhi (2).png
```

Comment:
it would be nice if you rename your image file name to just `abhi.png` for simplicity

Reply:
Changes Done for simplicity